### DISCUSSION TITLE
✅ Link in column

### DISCUSSION THREAD

**Discussion by cassiomurilo** [elementor/elementor#22389] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like to insert link in column in order to improve user usability.

I just solve it with another plugin or with code. Because it's something simple, it could be as a native resource.

![image](https://github.com/elementor/elementor/assets/27818524/16290308-3902-4c61-8b3a-2c7631eb6d71)

### Describe the solution you'd like

A simple solution is to reproduce what you already have in titles and other similar widgets

![image](https://github.com/elementor/elementor/assets/27818524/39d20d08-2759-453f-81df-74880fe4d47a)

**Comment by nicholaszein** [comment]
Hi @cassiomurilo!

We have great news! The feature you requested is already available in the Container, released on version 3.6.0 back in March, 2022.

We suggest you enable [Containers](https://elementor.com/help/what-is-a-container/) in your site if you haven't already, and [start benefitting from this feature](https://elementor.com/help/how-do-i-create-a-container/). You can also [Convert existing Sections and Columns into Containers](https://elementor.com/help/convert-existing-sections-to-containers/).

You can learn more about how to make Containers clickable it in the link below:
https://elementor.com/help/making-a-container-clickable/

Best regards!
